pcapfile/test/savefile_test.py

```diff
@@ -106,6 +106,8 @@ def test_basic_import(self):
         self.assertTrue(self.capfile.valid, 'invalid capture file')
         self.assertEqual(len(self.capfile.packets), 23,
                          'wrong number of packets!')
+        self.assertEqual(self.capfile.__length__(), 23,
+                         '__length__ not reporting correct number of packets')
```

Comment:
I am not sure what you mean @kisom. I think lines 107 and 108 do that don't they? Can you elaborate?